ast/parser.go

```diff
@@ -1956,12 +1957,20 @@ func (p *Parser) validateDefaultRuleValue(rule *Rule) bool {
 }
 
 type rawAnnotation struct {
-	Scope   string                `json:"scope"`
-	Schemas []rawSchemaAnnotation `json:"schemas"`
+	Scope            string                `yaml:"scope"`
```

Comment:
For some reason, `related_resources` didn't get unmarshaled because of the `_` when using `json`. With `yaml` it works. Don't see any reason for why this should be the case, though; but as we expect the metadata to be YAML, I don't think this change comes with any harm.

Reply:
Perhaps we could add a comment? I'm sure I won't remember the reason for this in a month or two 😅